Sources/Controllers/2.0/OAEditFavoriteViewController.mm

```diff
@@ -928,10 +925,30 @@ - (nonnull UITableViewCell *) tableView:(nonnull UITableView *)tableView cellFor
 - (void)tableView:(UITableView *)tableView willDisplayCell:(UITableViewCell *)cell forRowAtIndexPath:(NSIndexPath *)indexPath
  {
      NSDictionary *item = _data[indexPath.section][indexPath.row];
-     if ([cell isKindOfClass:OAFolderCardsCell.class])
-         [((OAFolderCardsCell *)cell) scrollToItemIfNeeded:(int)[item[@"selectedValue"] intValue]];
-     else if ([cell isKindOfClass:OAPoiTableViewCell.class])
-         [((OAPoiTableViewCell *)cell) updateContentOffset];
+     NSString *type = item[@"type"];
+     if ([type isEqualToString:kFolderCardsCell])
+     {
+         OAFolderCardsCell *folderCell = (OAFolderCardsCell *)cell;
+         if (!_scrollCellsState.values[kFolderCardsCell])
+         {
+             _scrollCellsState.values[kFolderCardsCell] = [NSValue valueWithCGPoint:[folderCell calculateOffset:(NSInteger)[item[@"selectedValue"] integerValue]]];
+         }
+         folderCell.cellTag = kFolderCardsCell;
+         folderCell.state = _scrollCellsState;
```

Comment:
You do not need to set the state in willDisplayCell, but rather only when. a new cell is being created in cellForRow.